### ISSUE TITLE
export onnx abnormally

### ISSUE THREAD

**Issue by zzy444626905** [ultralytics/yolov5#8389]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

export torchscript normally
export onnx abnormally and no error message, file named last.onnx does not exist

(yolov5) D:\yolov5>python export.py --weights runs/train/exp/weights/last.pt --data data/xfl.yaml --include onnx torchscript

export: data=data/xfl.yaml, weights=['runs/train/exp/weights/last.pt'], imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['onnx', 'torchscript']
YOLOv5  v6.1-259-ga2a1ed2 Python-3.7.11 torch-1.9.0 CPU

Fusing layers...
Model summary: 280 layers, 12308200 parameters, 0 gradients, 16.2 GFLOPs

PyTorch: starting from runs\train\exp\weights\last.pt with output shape (1, 25500, 6) (24.0 MB)

TorchScript: starting export with torch 1.9.0...
TorchScript: export success, saved as runs\train\exp\weights\last.torchscript (47.5 MB)

ONNX: starting export with onnx 1.10.1...
runs\train\exp\weights\last.onnx

### Environment

YOLOv5  v6.1-259-ga2a1ed2 Python-3.7.11 torch-1.9.0 CPU
os: win10
conda python3.8

### Minimal Reproducible Example

python export.py --weights runs/train/exp/weights/last.pt --data data/xfl.yaml --include onnx torchscript

### Additional

nothing

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by zzy444626905** [comment]
I have solved this problem...

**Comment by glenn-jocher** [comment]
@zzy444626905 glad to hear you've resolved the issue! If you encountered a specific resolution that could benefit others, feel free to share it. Your input could be quite helpful to the YOLOv5 community. If you have any further questions or need assistance with anything else, don't hesitate to let us know.
